### PR TITLE
Sex branch merge

### PR DESCRIPTION
**Pull request by lastDespair**
contains some of my test modules you might not want but go cry about it

### CODE REVIEW

File: StateFarmClient.js [hunk 1]
```diff
@@ -686,11 +686,13 @@ sniping and someone sneaks up on you
         ]);
             initModule({ location: tp.combatTab.pages[0], title: "Aimbot", storeAs: "aimbot", bindLocation: tp.combatTab.pages[1], defaultBind: "V", });
             initFolder({ location: tp.combatTab.pages[0], title: "Aimbot Options", storeAs: "aimbotFolder", });
+                initModule({ location: tp.aimbotFolder, title: "Force target refresh", storeAs: "targetRefresh", bindLocation: tp.combatTab.pages[1], enableConditions: [["aimbot", true]], }); //I hate this but whatever
```
Comment (VillainsRule) [inline StateFarmClient.js:689]: 1. change name to TargetRefresh or whatever
2. i don't see why we can't just do this all the time

File: StateFarmClient.js [hunk 2]
```diff
@@ -686,11 +686,13 @@ sniping and someone sneaks up on you
...
                 initModule({ location: tp.aimbotFolder, title: "TargetVisible", storeAs: "aimbotVisibilityMode", bindLocation: tp.combatTab.pages[1], dropdown: [{ text: "Disabled", value: "disabled" }, { text: "Prioritise Visible", value: "prioritise" }, { text: "Only Visible", value: "onlyvisible" }], defaultValue: "disabled", enableConditions: [["aimbot", true]] });
                 tp.aimbotFolder.addSeparator();
                 initModule({ location: tp.aimbotFolder, title: "ToggleRM", storeAs: "aimbotRightClick", bindLocation: tp.combatTab.pages[1], enableConditions: [["aimbot", true]], });
                 initModule({ location: tp.aimbotFolder, title: "SilentAim", storeAs: "silentAimbot", bindLocation: tp.combatTab.pages[1], enableConditions: [["aimbot", true]], });
+                initModule({ location: tp.aimbotFolder, title: "SemiSilent", storeAs: "aimbSemiSilent", bindLocation: tp.combatTab.pages[1], enableConditions: [["aimbot", true], ["silentAimbot", true]], });
```
Comment (VillainsRule) [inline StateFarmClient.js:695]: what is semi silent supposed to do

File: StateFarmClient.js [hunk 3]
```diff
@@ -720,6 +722,8 @@ sniping and someone sneaks up on you
 `Sorry! No guide yet!`},
         ]);
             initModule({ location: tp.renderTab.pages[0], title: "PlayerESP", storeAs: "playerESP", bindLocation: tp.renderTab.pages[1], });
+            initModule({ location: tp.renderTab.pages[0], title: "pPredESP", storeAs: "predictionESP", bindLocation: tp.renderTab.pages[1], });
```
Comment (VillainsRule) [inline StateFarmClient.js:725]: better names please

File: StateFarmClient.js [hunk 4]
```diff
@@ -2682,8 +2692,10 @@ z-index: 999999;
             ESPArray.push([object, tracerLines, box, target]);
         };
         object.tracerLines.setVerticesData(L.BABYLON.VertexBuffer.PositionKind, [crosshairsPosition.x, crosshairsPosition.y, crosshairsPosition.z, newPosition.x, newPosition.y, newPosition.z]);
-        object.tracerLines.color = new L.BABYLON.Color3(...color);
-        object.box.color = new L.BABYLON.Color3(...color);
+        //if(type!="pPredESP"){
```
Comment (VillainsRule) [inline StateFarmClient.js:2695]: remove comments

**Review by VillainsRule — CHANGES_REQUESTED** [review]
fix cod it suck

File: StateFarmClient.js (hunk 1 above)
Comment (lastDespair) [reply StateFarmClient.js:689]: idk I didnt write this initially, legit cheat test just fucked with targets staying targets after leaving minAngle

File: StateFarmClient.js (hunk 2 above)
Comment (lastDespair) [reply StateFarmClient.js:695]: changes looks at the location it just shot at

File: StateFarmClient.js (hunk 3 above)
Comment (lastDespair) [reply StateFarmClient.js:725]: shut up the name is perfect

File: StateFarmClient.js (hunk 4 above)
Comment (lastDespair) [reply StateFarmClient.js:2695]: oh lol why that still in there

**Comment by onlypuppy7** [comment]
lets merge and then fix problems later

**Pull request merged**